var/spack/repos/builtin/packages/nvpl-scalapack/package.py

```diff
@@ -0,0 +1,65 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class NvplScalapack(Package):
+    """NVPL ScaLAPACK (NVIDIA Performance Libraries ScaLAPACK)."""
+
+    homepage = "https://docs.nvidia.com/nvpl/latest/scalapack/index.html"
+    url = (
+        "https://developer.download.nvidia.com/compute/nvpl/redist/nvpl_scalapack/"
+        "linux-sbsa/nvpl_scalapack-linux-sbsa-0.2.1-archive.tar.xz"
+    )
+
+    maintainers("RMeli")
+
+    version("0.2.1", sha256="dada4d1ecf044d90609b9e62750b383d11be9b22c87e109414bcc07dce3c83c9")
+
+    provides("scalapack")
+
+    variant("ilp64", default=False, description="Force 64-bit Fortran native integers")
+
+    depends_on("nvpl-blas +ilp64", when="+ilp64")
+    depends_on("nvpl-blas ~ilp64", when="~ilp64")
+    depends_on("nvpl-lapack +ilp64", when="+ilp64")
+    depends_on("nvpl-lapack ~ilp64", when="~ilp64")
+    depends_on("mpi")
+
+    requires("target=armv8.2a:", msg="Any CPU with Arm-v8.2a+ microarch")
+
+    conflicts("%gcc@:7")
+    conflicts("%clang@:13")
+
+    def url_for_version(self, version):
+        """Spack can't detect the verion in the URL above"""
```

Comment:
```suggestion
        """Spack can't detect the version in the URL above"""
```

This comment can also be dropped...up to you